src/interface/report/PatchChecker.js

```diff
@@ -52,7 +53,13 @@ class PatchChecker extends React.PureComponent {
     const reportTimestamp = report.start;
     const reportDate = new Date(report.start).toString();
 
-    if (reportTimestamp >= LATEST_PATCH || this.continue) {
+    const expansionStartPatch = PATCHES[0];
+    // report's patch is the last patch where the reports timestamp is higer than the patch timestamp
+    // find works on first item so we reverse the array to start from the end
+    // use slice to copy the array first so we don't mutate PATCHES
+    const reportPatch = PATCHES.slice().reverse().find(patch => reportTimestamp > patch.timestamp);
```

Comment:
Rather than doing a reverse (which relies on the the array being updated in a specific order), maybe a sort instead?